lightning/src/ln/channel.rs

```diff
@@ -928,9 +929,17 @@ impl<Signer: Sign> Channel<Signer> {
 			}
 		}
 
+		// Also update `ChannelConfig::announced_channel` to make sure we still allow users to
+		// downgrade versions after the fact.
+		//
+		// TODO: This can be removed once we prevent downgrades to v0.0.107.
```

Comment:
Not 100% sure what exactly you mean here with "prevent downgrades to v0.0.107". Do you basically mean "when we're ready to break backwards compat with versions prior to v0.0.107"?

Reply:
Correct, I can reword to your suggestion if it's not clear as is.